learning/views.py

```diff
@@ -23,6 +23,15 @@ def display_character(request, character_pk):
     except ObjectDoesNotExist:
         character = Character.objects.filter(pk__gt=character_pk).first()
         return redirect('display_character', character_pk=character.pk)
+    try:
+        status = request.session['is_learning']
+    except KeyError:
+        status = False
+    if status == True:
+        learning_process(request)
+    else:
+        request.user.last_study_vocab_count = 0
+        start_learning(request, 10)
```

Comment:
here `start_learning()` should be called in the `urls.py` if user clicks start learning in index page. The reason the start learning button directs the user here is because the learning logic isn't finished...